app/models/container_node.rb

```diff
@@ -115,4 +117,14 @@ def external_logging_path
     index = ".operations.*"
     EXTERNAL_LOGGING_PATH % {:index => index, :query => query}
   end
+
+  def disconnect_inv
+    return if ems_id.nil?
+    _log.info "Disconnecting Node [#{name}] id [#{id}] from EMS [#{ext_management_system.name}]" \
+    "id [#{ext_management_system.id}] "
+    self.old_ems_id = ems_id
+    self.ext_management_system = nil
+    self.deleted_on = Time.now.utc
+    save
+  end
```

Comment:
@simon3z Note we keep entities created by this node like `hardware` and `computer_system`. i cant think of a place we might want to exclude them (reasons we would want to implement disconnection for them) but we should keep that in mind